ucla-prrla/src/app/detail-item/detail-item.component.ts

```diff
@@ -110,6 +112,13 @@ export class DetailItemComponent implements OnInit {
 
         this.testService.getItemById(this.id).subscribe(data => {
             this.item = data;
+
+            if (data instanceof Object) {
+                this.title.setTitle(`${data.title} | PRL`);
+            } else {
+                this.title.setTitle("FUBAR | PRL");
```

Comment:
Cute. But at the risk of sounding like a bluenose, maybe use a less... incendiary term? I suspect most people aren't as clueless as Upham in _Saving Private Ryan_.

Reply:
hmm, my search pattern for swear words needs updating to account for abbreviations. Anyway, changed

Reply:
@markmatney out of curiosity, do you have a git hook, or something, for detecting swear words before push or was that just a throwaway comment?

Reply:
No, just kidding. LOL. Although that's a good idea